### ISSUE TITLE
Pure dark theme mode

### ISSUE THREAD

**Issue by MajorCooke** [sirjuddington/SLADE#1188]
It'd be appreciated if there could be a pure dark theme variant that affects the entire program versus just the coding box. Currently I have to have the program use the default light theme with the code because it creates strain on my eyes from glancing at the dark code, from the bright surrounding program all around it.

**Comment by sirjuddington** [comment]
This request has come up a lot lately. wxWidgets uses standard windows controls which can't be 'themed' at all, so unfortunately it's not possible without rewriting the entire UI using something themable in Windows (eg. Qt), which would take a very long time.

**Comment by BinarryCode** [comment]
> wxWidgets uses standard windows controls which can't be 'themed' at all

Would it be possible now that [wxWidgets can detect darkmode in the Windows registry](https://github.com/wxWidgets/wxWidgets/pull/22020)?

**Comment by sirjuddington** [comment]
Nope, that is purely for detecting if 'dark mode' is enabled in windows (which only really affects UWP apps and some custom-themed things like explorer and web browsers), the issue is that windows doesn't provide any dark theme controls to use at all.

**Comment by sirjuddington** [comment]
Looks like there may be hope for this in the future, the next version of wxWidgets appears to be adding 'experimental' dark mode support in Windows(10 20H1+ only): https://docs.wxwidgets.org/latest/classwx_app.html#af8c93d7e3345e62a58325f3ab1d158d6

**Comment by Chonkblonk** [comment]
Looking forward to maybe seeing this realized some day. Sometimes I've maybe got a headache and less bright lights in my face the better, and some days are just so dark around here that even turning on all the lights only helps so much.

Very cool program at any rate, it's a lot like XWE which I was using as a teen, except it's stable and has better features.

**Comment by mknepper** [comment]
I'm definitely excited to see this being added. I just installed Slade and noticed there is no dark mode for it. Hopefully we'll see it soon. If I knew how to do it, I would make the commit myself, but I have no idea how wxWidgets works.
